TESTS/psa-compliance/test_s001/test_s001.c

```diff
@@ -0,0 +1,168 @@
+/** @file
+ * Copyright (c) 2019, Arm Limited or sst affiliates. All rights reserved.
+ * SPDX-License-Identifier : Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+**/
+#define ITS_TEST
```

Comment:
Is is possible to maintain test specific mbed_lib.json to pass ITS_TEST macro? 
Idea is to avoid test editing.

Reply:
No